ui/packages/consul-ui/app/templates/dc/peers/index.hbs

```diff
@@ -0,0 +1,55 @@
+<Route @name={{route}} as |route|>
+  <AppView>
+    <BlockSlot @name="header">
+      <h1>
+        <route.Title @title="Peers" />
+      </h1>
+    </BlockSlot>
+    <BlockSlot @name="toolbar">
+      <Peerings::Search @search={{this.filter}} @onSearch={{this.handleSearchChanged}} />
```

Comment:
We have a search component that we use for all our searching, which I think I'll need to replace this with once this is handed over. So if you wanted to copy in the partition one in here and rename then great, but thats on my list for things to do once this PR is in, so just a heads up that I'll probably replace this with our component once this is merged in.

I'm kinda looking for the comment you had on the Firefox issue which is something you could help me with also though, I'll comment elsewhere once I see that.

Reply:
The existing component is pretty tightly coupled to how the custom router dsl sets up its search-properties. The peerings search isn't filterable by anything yet but will only search by name. That's why I decided to go with a custom implementation. Not sure if the crud stuff you are working on needs a new search component or adds additional search capabilities but feel free to change whatever you feel is necessary.